plugins/usbdmx/DMXCProjectsNodleU1Factory.cpp

```diff
@@ -33,22 +33,53 @@ namespace usbdmx {
 const uint16_t DMXCProjectsNodleU1Factory::VENDOR_ID = 0x16d0;
 const uint16_t DMXCProjectsNodleU1Factory::PRODUCT_ID = 0x0830;
 
+// Those IDs are not officially registered or "free for everyone"
+// but there seems to be one clone using them.
+// See also: https://github.com/mcallegari/qlcplus/blob/3b69452d0679333c6e60ff0928c20f2107ccc81f/plugins/hid/hiddmxdevice.h#L34
+const uint16_t DMXCProjectsNodleU1Factory::VENDOR_ID_FX5 = 0x16c0;
+const uint16_t DMXCProjectsNodleU1Factory::PRODUCT_ID_FX5 = 0x088b;
+
 bool DMXCProjectsNodleU1Factory::DeviceAdded(
     WidgetObserver *observer,
     libusb_device *usb_device,
     const struct libusb_device_descriptor &descriptor) {
-  if (descriptor.idVendor != VENDOR_ID || descriptor.idProduct != PRODUCT_ID) {
+  OLA_DEBUG << "Factory, DeviceAdded";
+  if (
+    ((descriptor.idVendor != VENDOR_ID) &&
+    (descriptor.idVendor != VENDOR_ID_FX5)) ||
+    ((descriptor.idProduct != PRODUCT_ID) &&
+    (descriptor.idProduct != PRODUCT_ID_FX5))
+    ) {
     return false;
   }
 
-  OLA_INFO << "Found a new Nodle U1 device";
+  OLA_INFO << "Found a new Nodle U1 or clone device";
   ola::usb::LibUsbAdaptor::DeviceInformation info;
   if (!m_adaptor->GetDeviceInfo(usb_device, descriptor, &info)) {
```

Comment:
This fails for the hotplug-case of the RP2040-based dongle:
```
plugins/usbdmx/DMXCProjectsNodleU1Factory.cpp:56: Found a new Nodle U1 or clone device
libs/usb/LibUsbAdaptor.cpp:53: libusb_get_string_descriptor_ascii failed: LIBUSB_ERROR_BUSY
```
I'd need to re-check when the "detach host driver" step should be done and when it is actually done